### PR TITLE
Fixes response of agent_simulator for active-response config request

### PR DESCRIPTION
**Pull request by RamosFe**
| Issue |
|---------|
| https://github.com/wazuh/wazuh-qa/issues/1266|

# Description
Fixes the response of the `agent simulator` for the active-response configuration messages.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,6 +16,10 @@ All notable changes to this project will be documented in this file.
 
 - Enhance the PR template. ([#4881](https://github.com/wazuh/wazuh-qa/pull/4881)) \- (Framework)
 
+### Fixed
+
+- Fixed agent_simulator response for active-response configuration commands. ([#4895](https://github.com/wazuh/wazuh-qa/pull/4895)) \- (Framework)
```
Comment (GGP1) [inline CHANGELOG.md:21]: Include this section below `4.8.2`

**Review by GGP1 — CHANGES_REQUESTED** [review]
Changes look good to me, but the release type is `patch`, so we should point to `4.8.2` instead of `master`.

File: CHANGELOG.md (hunk 1 above)
Comment (RamosFe) [reply CHANGELOG.md:8]: Changed in commit 57dd3a9e6e087d489a356d53f106b1c99bb814d1. Thanks!

**Review by GGP1 — APPROVED** [review]
LGTM

**Review by Selutario — APPROVED** [review]
LGTM.

**Review by damarisg — COMMENTED** [review]
It's ok. However, I suggest adding runs of all suites to make sure nothing breaks. Apparently it seems not, but our agent_simulator affects different tests and is sometimes unstable. Just to make sure.

File: deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py [hunk 2]
```diff
@@ -540,7 +540,11 @@ def process_command(self, sender, message_list):
                     sender.send_event(self.create_event(f'#!-req {req_code} '
                                                         f'{{"error":0, "message":"ok", "data":[]}} '))
         elif command == 'getconfig':
-            response_json = '{"client":{"config-profile":"centos8","notify_time":10,"time-reconnect":60}}'
+            if "active-response" in message_list:
+                response_json = '{{"active-response":{"disabled":"no"}}'
```
Comment (Selutario) [inline deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py:544]: This dict is malformed, it was raising this error:

> `GET /agents/040/config/com/active-response`

```JSON
{
  "title": "Wazuh Internal Error",
  "detail": "JSON couldn't be loaded",
  "dapi_errors": {
    "master-node": {
      "error": "JSON couldn't be loaded",
      "logfile": "WAZUH_HOME/logs/api.log"
    }
  },
  "error": 3036
}
``` 

```suggestion
                response_json = '{"active-response":{"disabled":"no"}}'
```

**Review by Selutario — CHANGES_REQUESTED** [review]

File: deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py (hunk 2 above)
Comment (RamosFe) [reply deps/wazuh_testing/wazuh_testing/tools/agent_simulator.py]: Fixed in c4cf2da90dca4c5e4c554a47187ca8efaa82a1c2.

**Review by Selutario — APPROVED** [review]

**Comment by damarisg** [comment]
The results of remoted are found [here](https://github.com/wazuh/wazuh-qa/issues/1266#issuecomment-2065347028). Are we sure this change has no negative impact on other tests? Could you run all suites to verify it?

**Pull request merged**